internal/constants/constants.go

```diff
@@ -211,7 +217,7 @@ func State() string {
 
 var (
 	// versionInfo is the version of a binary. Left as a separate variable to allow override during build.
-	versionInfo = "0.0.0"
+	versionInfo = "2.8.0"
```

Comment:
This should not be changed. To force a specific version in your CLI you can change the `version.txt` file in the repo's root folder.

Reply:
Oops! That was accidental